### PR TITLE
Re-implement IRC channel history/logs

### PR DESCRIPTION
**Pull request by raucao**
Re-implements the IRC channel history, based on the new public logs meta documents, so we don't have to search for the most recent log files anymore.

### CODE REVIEW

**Comment by raucao** [comment]
CodeClimate broke our repo page, so I cannot see what it's complaining about. Sent them an email about it.

**Review by galfert — CHANGES_REQUESTED** [review]
Left a comment, the rest looks ok to me.

File: app/services/coms.js [hunk 1]
```diff
@@ -288,57 +290,65 @@ export default class ComsService extends Service {
       this.storage.saveChannel(channel);
     }
 
-    if (channel.isLogged) {
-      this.loadLastMessages(channel, moment.utc(), 2)
-          .catch(() => { /* TODO nothing to do here? */ });
+    // TODO Check other message archives first, depending on protocol
+    // (e.g. MAM for XMPP)
+    if (account.protocol === 'IRC') {
+      this.loadLastMessagesFromKosmosArchives(channel, options.isLogged);
```
[the remark this answers is not in the document]
Comment (raucao) [reply app/services/coms.js:296]: Good catch, thanks!

**Review by galfert — APPROVED** [review]
👍

**Pull request merged**